android/res/layout/item_add_street.xml

```diff
@@ -3,7 +3,7 @@
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
-  android:layout_height="@dimen/height_item_oneline"
```

Comment:
Is this identifier used somewhere else? If not, please clean it up.

Reply:
Same for this 
![image](https://user-images.githubusercontent.com/46300244/230201030-b382ae4c-dad6-41c5-b763-c8e2879152f9.png)